### PR TITLE
🏛️ Add routes and templates to add organizations

### PR DESCRIPTION
**Pull request by FedericoDM**
## Describe your changes

- Adding route for admin to add organizations and a user tied to it
- Adding util function to create a temp password
- Adding templates for adding orgs

## Checklist before requesting a review
- [x] pre-commit run --all-files (run before pushing)
- [x] pytest if applicable
- [x] Link issue
- [x] Update relevant documentation if applicable: doc strings, readme, poetry.

```commandline
HERE IS SOME COMMAND LINE OUTPUT
```

### CODE REVIEW

**Comment by FedericoDM** [comment]
@aaronhaefner this PR has the routes and templates to add organizations and a user, along with a simple function to generate temp passwords.

File: new_arrivals_chi/app/main.py [hunk 1]
```diff
@@ -517,6 +530,97 @@ def edit_organization():
...
+            # Create the user with the provided email and a default password
+            new_user = create_user(email, temp_pwd)
+
+            # Update the user with the new organization
+            user = User.query.get(new_user.id)
+            if user:
+                user.organization_id = new_org_id
+                db.session.commit()
+            else:
+                raise Exception("User not found")
```
Comment (MadeleineKRoberts) [inline new_arrivals_chi/app/main.py:608]: we probably want to do a try catch here so that the exception doesnt crash the app

**Review by MadeleineKRoberts — CHANGES_REQUESTED** [review]
Try Catch needed to prevent crash

File: new_arrivals_chi/app/main.py (hunk 1 above)
Comment (FedericoDM) [reply new_arrivals_chi/app/main.py]: Done!

**Review by MadeleineKRoberts — APPROVED** [review]
thank you for the change!

**Pull request merged**